application/models/item.go

```diff
@@ -123,11 +123,18 @@ func (i *Item) Update(ctx context.Context) error {
 		}
 	}
 
-	if oldItem.CoverageAmount != i.CoverageAmount && i.CoverageStatus == api.ItemCoverageStatusApproved &&
-		!i.CoverageEndDate.Valid {
-		amount := i.calculatePremiumChange(time.Now().UTC(), oldItem.CoverageAmount)
-		if err := i.CreateLedgerEntry(Tx(ctx), LedgerEntryTypeCoverageChange, amount); err != nil {
-			return err
+	if oldItem.CoverageAmount != i.CoverageAmount {
+		if i.CoverageStatus == api.ItemCoverageStatusApproved {
+			if oldItem.CoverageAmount < i.CoverageAmount {
+				err := errors.New("item coverage amount cannot be increased")
+				return api.NewAppError(err, api.ErrorItemCoverageAmountCannotIncrease, api.CategoryUser)
+			}
+			if !i.CoverageEndDate.Valid {
```

Comment:
If you wanted to reduce the depth of if statements you could remove `if oldItem.CoverageAmount != i.CoverageAmount {` and use:
```suggestion
			if !i.CoverageEndDate.Valid && oldItem.CoverageAmount < i.CoverageAmount {
```

Reply:
Thanks. I pushed that change to develop